.github/workflows/pr-environment.yml

```
@@ -0,0 +1,59 @@
+        with:
+          node-version: '20'
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Remove SST
+        run: npx sst remove --stage ${{ github.event.pull_request.number }}
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: "us-east-1" 
```

yamllint

[error] trailing spaces

(trailing-spaces)